lib/rules/no-curly-component-invocation.js

```diff
@@ -43,7 +43,7 @@ const BUILT_INS = [
   'in-element',
 ];
 
-const ALWAYS_CURLY = ['yield'];
+const ALWAYS_CURLY = ['yield', 'app-version' /* from ember-cli-app-version */];
```

Comment:
I think this belongs in the list of built ins above, not this ALWAYS_CURLY listing.  ALWAYS_CURLY is about things that **must** be invoked in curlies, but this is a helper and can definitely be used in subexpression's etc.

Reply:
yeah, it was my first idea, but rule failing in this case (now it's fixed)